photon-api/src/main/scala/com/linkedin/photon/ml/evaluation/RMSEEvaluator.scala

```diff
@@ -31,7 +31,9 @@ object RMSEEvaluator extends SingleEvaluator {
    */
   override def evaluate(scoresAndLabelsAndWeights: RDD[(Double, Double, Double)]): Double = {
 
-    val squaredLoss = SquaredLossEvaluator.evaluate(scoresAndLabelsAndWeights)
+    // squared loss is calculated as 1/2 (z - y)^2 for mathematical convenience
+    // however, RMSE should be using (z - y)^2 for calculation, hence the multiplying by 2 here:
```

Comment:
Please capitalize the first word in each comment

Reply:
fixed.